docs/articles/certificates.md

```diff
@@ -1,17 +0,0 @@
-<!--[metadata]>
```

Comment:
Yup, it's in place already; see https://github.com/docker/docker/pull/19511/files#diff-1ae60c50f995a90b9a434c5e0fd3f004R3;

    aliases = ["/articles/certificates/"]